### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by LuccaWang1**
This PR added `title`, `description`, `table of contents`, `author` sections to the README. 

IMG_9329

### CODE REVIEW

**Comment by CrisKP** [comment]
Adding a general comment here

**Comment by LuccaWang1** [comment]
> Adding a general comment here

Hi

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# practicing-git
\ No newline at end of file
+# practicing-git project
+
+## Project Description
+This project is for practicing git, creating new branches - one for each feature, pull requests, reviews, approvals, and pushing.
+
+## Table of Contents 
+- Author
+- Technologies Used
```
Comment (CrisKP) [inline README.md]: I think this should be reversed

```suggestion
- Technologies Used
- Author
```

**Review by CrisKP — APPROVED** [review]
Looks good to go ✨

**Pull request merged**